class_customer.py

```diff
@@ -0,0 +1,64 @@
+# Placeholder class
+class User:
+    def __init__(self, fname, lname):
+        self.fname = fname
+        self.lname = lname
+        self.login = None
+        self.password = None
+
+
+class Customer(User):
+    role = "customer"
+
+    def __init__(self, fname, lname):
+        super().__init__(fname, lname)
+        self.cart = {}  # initializes dictionary for the cart, {book : stock_quantity}
+
+    @staticmethod
+    def view_book(book):  # method to display information of the book
+        print(f"Name: {book.name}",
+              f"Author: {book.author}",
+              f"Price: {book.price}",
+              f"Stock quantity: {book.stock_quantity}")
+
+    def add_to_cart(self, book, quantity):
+        if book in book.all_books:  # 'book.all_books' is a placeholder for database with books
+
+            if book.stock_quantity >= quantity:  # check if the requested amount of books is available
+
+                if book not in self.cart:  # check if the book hasn't been added to the cart before
+                    self.cart.update({book: quantity})  # could be 'book.name' instead of book object
+                    book.stock_quantity = - quantity
```

Comment:
I think this line would negate the quantity number instead of substracting from the stock quantity. Anyway I think that line would change later so it's not so important